Traceback (most recent call last):
  File "/Users/enaj/.virtualenvs/p39/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/enaj/.virtualenvs/p39/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/enaj/.virtualenvs/p39/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/enaj/.virtualenvs/p39/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/enaj/.virtualenvs/p39/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/enaj/.virtualenvs/p39/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/enaj/.virtualenvs/p39/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
start_scope()
from matplotlib.image import imread
img = (1-imread('brian.png'))[::-1, :, 0].T
num_samples, N = img.shape
ta = TimedArray(img, dt=1*ms) # 228
A = 1.5
tau = 2*ms
eqs = '''
dv/dt = (A*ta(t, i)-v)/tau+0.8*xi*tau**-0.5 : 1
'''
G = NeuronGroup(N, eqs, threshold='v>1', reset='v=0', method='euler')
M = SpikeMonitor(G)
run(num_samples*ms)
plot(M.t/ms, M.i, '.k', ms=3)
xlim(0, num_samples)
ylim(0, N)
xlabel('Time (ms)')
ylabel('Neuron index');
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
Input [0;32mIn [16][0m, in [0;36m<cell line: 3>[0;34m()[0m
[1;32m      1[0m start_scope()
[1;32m      2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mimage[39;00m [38;5;28;01mimport[39;00m imread
[0;32m----> 3[0m img [38;5;241m=[39m ([38;5;241m1[39m[38;5;241m-[39m[43mimread[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mbrian.png[39;49m[38;5;124;43m'[39;49m[43m)[49m)[::[38;5;241m-[39m[38;5;241m1[39m, :, [38;5;241m0[39m][38;5;241m.[39mT
[1;32m      4[0m num_samples, N [38;5;241m=[39m img[38;5;241m.[39mshape
[1;32m      5[0m ta [38;5;241m=[39m TimedArray(img, dt[38;5;241m=[39m[38;5;241m1[39m[38;5;241m*[39mms) [38;5;66;03m# 228[39;00m

File [0;32m~/.virtualenvs/p39/lib/python3.9/site-packages/matplotlib/image.py:1560[0m, in [0;36mimread[0;34m(fname, format)[0m
[1;32m   1558[0m                 response [38;5;241m=[39m io[38;5;241m.[39mBytesIO(response[38;5;241m.[39mread())
[1;32m   1559[0m             [38;5;28;01mreturn[39;00m imread(response, [38;5;28mformat[39m[38;5;241m=[39mext)
[0;32m-> 1560[0m [38;5;28;01mwith[39;00m [43mimg_open[49m[43m([49m[43mfname[49m[43m)[49m [38;5;28;01mas[39;00m image:
[1;32m   1561[0m     [38;5;28;01mreturn[39;00m (_pil_png_to_float_array(image)
[1;32m   1562[0m             [38;5;28;01mif[39;00m [38;5;28misinstance[39m(image, PIL[38;5;241m.[39mPngImagePlugin[38;5;241m.[39mPngImageFile) [38;5;28;01melse[39;00m
[1;32m   1563[0m             pil_to_array(image))

File [0;32m~/.virtualenvs/p39/lib/python3.9/site-packages/PIL/ImageFile.py:104[0m, in [0;36mImageFile.__init__[0;34m(self, fp, filename)[0m
[1;32m    100[0m [38;5;28mself[39m[38;5;241m.[39mdecodermaxblock [38;5;241m=[39m MAXBLOCK
[1;32m    102[0m [38;5;28;01mif[39;00m isPath(fp):
[1;32m    103[0m     [38;5;66;03m# filename[39;00m
[0;32m--> 104[0m     [38;5;28mself[39m[38;5;241m.[39mfp [38;5;241m=[39m [38;5;28;43mopen[39;49m[43m([49m[43mfp[49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mrb[39;49m[38;5;124;43m"[39;49m[43m)[49m
[1;32m    105[0m     [38;5;28mself[39m[38;5;241m.[39mfilename [38;5;241m=[39m fp
[1;32m    106[0m     [38;5;28mself[39m[38;5;241m.[39m_exclusive_fp [38;5;241m=[39m [38;5;28;01mTrue[39;00m

[0;31mFileNotFoundError[0m: [Errno 2] No such file or directory: 'brian.png'
FileNotFoundError: [Errno 2] No such file or directory: 'brian.png'

